main_page.php

```diff
@@ -16,13 +32,20 @@
 </head>
 <body>
     <?php
-        foreach ($recipes as $recipe) {
-            echo '<section class= "recipe_card">';
-            echo '<h2>' . $recipe['recipe'] . '<h2>';
-            echo '<h3>Cuisine: ' . $recipe['cuisine'] . '</h3>';
-            echo '<h3>Time (mins): ' . $recipe['time'] . '</h3>';
-            echo '<a href=' . $recipe['link'] .'>Link to recipe</a>';
-            echo '</section>';
+        /**
+         * Creates recipe cards which have a name as well as stats on cuisine, time and a link to the recipe
+         */
+        function createRecipeCards() {
+            foreach ($recipes as $recipe) {
+                echo '<section class= "recipe_card">';
+                echo '<h2>' . $recipe['recipe'] . '<h2>';
+                echo '<h3>Cuisine: ' . $recipe['cuisine'] . '</h3>';
+                echo '<h3>Time (mins): ' . $recipe['time'] . '</h3>';
+                echo '<a href=' . $recipe['link'] . '>Link to recipe</a>';
+                echo '</section>';
+            }
```

Comment:
What happens if there are no recipes? Displaying just a simple error message would do the trick!